src/client.ts

```diff
@@ -20,15 +21,24 @@ async function initBotConfig(client: Client, db: Db) {
     // Persist default configuration data
     const defaultChannelId =
```

Comment:
If we're getting more and more of these env variables, I think we'll want to handle validating their presence here

Reply:
Can you expand on what you mean by this? What kind of validation do you want to see?

Reply:
To clarify, what should the behavior be when these defaults aren't set? I think it's handled later for queue channel but I didn't see anything for the other two

Reply:
Ah I see, so default handling in the vote commands would satisfy this?

Reply:
Yes, it could be handled the same where where a prompt to set the config is sent

Reply:
Added some error handling in the vote subcommand implementation.